### PR TITLE
Add classic stylelint

### PR DESCRIPTION
**Pull request by NeOMakinG**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | Adding stylelint like new-theme inside the classic theme, great benefits from it : more maintainable, cleaner code, avoid bugs..
| Type?         | improvement
| Category?     | FO
| BC breaks?    | no
| Deprecations? | no
| How to test?  | This is these sort of PR where you need to check if nothing changed on FO, because this is mainly code refacto :(

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/20384)

### CODE REVIEW

**Review by matthieu-rolland — DISMISSED** [review]

File: themes/classic/_dev/css/components/products.scss [hunk 1]
```diff
@@ -335,72 +394,75 @@
...
     padding: 3.125rem 1.875rem;
+    background: $btn-warning-color;
     .divide-right span {
       display: inline-block;
       margin-bottom: 0.3125rem;
 
       @include media-breakpoint-down(sm) {
         display: block;
-        padding: 0 .5rem;
+        padding: 0 0.5rem;
```
Comment (Progi1984) [inline themes/classic/_dev/css/components/products.scss:405]: Normal ?

File: themes/classic/_dev/css/components/products.scss [hunk 2]
```diff
@@ -335,72 +394,75 @@
...
     }
   }
   .product-name {
-    color: $brand-primary;
     font-size: 1.125rem;
+    color: $brand-primary;
 
     @include media-breakpoint-down(sm) {
-      padding: 0 .5rem;
+      padding: 0 0.5rem;
```
Comment (Progi1984) [inline themes/classic/_dev/css/components/products.scss:433]: Normal ?

File: themes/classic/_dev/css/components/products.scss [hunk 3]
```diff
@@ -335,72 +394,75 @@
...
     }
   }
   .product-price {
-    color: $gray-dark;
     display: block;
+    color: $gray-dark;
     @include media-breakpoint-down(sm) {
-      padding: 0 .5rem;
+      padding: 0 0.5rem;
```
Comment (Progi1984) [inline themes/classic/_dev/css/components/products.scss:440]: Normal ?

File: themes/classic/_dev/css/components/products.scss [hunk 4]
```diff
@@ -335,72 +394,75 @@
...
     padding-left: $extra-large-space;
+    .btn {
+      margin-bottom: $small-space;
+    }
     p {
-      color: $gray-dark;
       display: flex;
       justify-content: space-between;
-      padding: 0 .5rem;
+      padding: 0 0.5rem;
```
Comment (Progi1984) [inline themes/classic/_dev/css/components/products.scss:452]: Normal ?

File: themes/classic/_dev/css/partials/_commons.scss [hunk 5]
```diff
@@ -459,5 +553,6 @@ input, textarea {
 .modal-dialog-centered {
   display: flex;
   align-items: center;
-  min-height: calc(100% - (30px * 2)); // 30px correspond to the default margin size from bootstrap 4.0.0-alpha.5 for modal-dialog class
+  // 30px correspond to the default margin size from bootstrap 4.0.0-alpha.5 for modal-dialog class
```
Comment (Progi1984) [inline themes/classic/_dev/css/partials/_commons.scss]:
```suggestion
  // 30px corresponds to the default margin size from bootstrap 4.0.0-alpha.5 for modal-dialog class
```

**Comment by NeOMakinG** [comment]
@Progi1984 I accepted your request, but it wasn't related to the PR !

**Review by Progi1984 — APPROVED** [review]

**Review by PierreRambaud — APPROVED** [review]

**Comment by ngodefroy** [comment]
Hello @NeOMakinG 

I have tested the PR and did not found any bug.
To do so I did some exploratory tests:

- Placing orders
- Editing customer address
- Checked all the pages under "Your account" (Information/Addresses/Order History/Credit slips/GDPR
- Contacting the store
- While Checking the console box to see if any error appeared

Thank you for your contribution

**Comment by matks** [comment]
Thanks @NeOMakinG for the PR and thanks @ngodefroy for the QA

**Pull request merged**